packages/twenty-docs/src/pages/rest-api.tsx

```diff
@@ -0,0 +1,120 @@
+import Layout from "@theme/Layout";
+import BrowserOnly from "@docusaurus/BrowserOnly";
+import React, { useEffect, useState } from "react";
+import { API } from '@stoplight/elements';
+import '@stoplight/elements/styles.min.css';
+import './graphql.css'
+import { parseJson } from "nx/src/utils/json";
+import { TbLoader2 } from "react-icons/tb";
+
+interface TokenFormPropsI {
```

Comment:
if possible use types and not interfaces per project convention
no need to suffix by I: type TokenFormProps